### ISSUE TITLE
KeyError: 'TOP'

### ISSUE THREAD

**Issue by mdavidsaver** [epics-base/ci-scripts#89]
The `TOP` magic variable is no longer being set early enough.  A regression in dffe32c23e8c54daa0f19a57b9de9bda015d73ea where I moved two too many lines...

```
python .ci/cue.py \
  --add-path "{TOP}/bin/{EPICS_HOST_ARCH}" \
  --add-path "{TOP}/bundle/usr/{EPICS_HOST_ARCH}/lib" \
 exec pvxinfo -D
```

After updating to 20f8e053931fdef8a9413cc6229286c5d9ed152f, now fails with:

```
Traceback (most recent call last):
  File "/home/runner/work/pvxs-dev/pvxs-dev/.ci/cue.py", line 1477, in <module>
    main(sys.argv[1:])
  File "/home/runner/work/pvxs-dev/pvxs-dev/.ci/cue.py", line 1473, in main
    args.func(args)
  File "/home/runner/work/pvxs-dev/pvxs-dev/.ci/cue.py", line 1354, in doExec
    setup_for_build(args)
  File "/home/runner/work/pvxs-dev/pvxs-dev/.ci/cue.py", line 770, in setup_for_build
    addpaths.append(path.format(**os.environ))
KeyError: 'TOP'
```

**Comment by mdavidsaver** [comment]
Should be fixed by 130e88b7095812da93e423c602651e30f39da11a.

**Comment by ralphlange** [comment]
Thanks!

Let me add a test first...
